### PR TITLE
Viz legend

### PR DESCRIPTION
**Pull request by tmushayahama**
### Issues

- https://github.com/geneontology/wc-gocam-viz/issues/37

### Changed

- Added the provides input for
- prettified other icons

cc @pkalita-lbl @kltm @vanaukenk @dustine32

### CODE REVIEW

File: src/components/gocam-legend/gocam-legend.scss [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
 .header {
   @include standard-var-styles(header);
+  text-align: center;
```
Comment (pkalita-lbl) [inline src/components/gocam-legend/gocam-legend.scss:31]: I know it seems like a minor thing, but I'd prefer to keep the base style as unopinionated as possible. If a client wants this text centered there is already a way for them to accomplish that, as illustrated here: https://github.com/geneontology/wc-gocam-viz/blob/5c373a9e46f77b8b623f7bab90670906987c4c60/src/styled.html#L46-L48

**Comment by tmushayahama** [comment]
@pkalita-lbl thanks, I have removed the styling

**Comment by tmushayahama** [comment]
@pkalita-lbl @kltm should we merge this?

**Comment by kltm** [comment]
For #37

**Review by pkalita-lbl — APPROVED** [review]
Looks good to me

**Comment by vanaukenk** [comment]
I've checked this on noctua-dev.

The legend now displays the 'provides input for' relation in my [test model](http://noctua-dev.berkeleybop.org/workbench/noctua-visual-pathway-editor/?model_id=gomodel%3A67299f1a00000071).

Noting, though, that the width of the relation in the pathway view still looks smaller than the other relations, and in bigger models, the lighter color of that relation can get lost as noted in:
https://github.com/geneontology/wc-gocam-viz/issues/37#issuecomment-2155227082
https://github.com/geneontology/wc-gocam-viz/issues/37#issuecomment-2169344977

We can deal with the display issue in separate work, as this PR does fix the legend, as requested.

**Pull request merged**